app/models/order_delivery_address.rb

```diff
@@ -0,0 +1,26 @@
+class OrderDeliveryAddress
+  include ActiveModel::Model
+  attr_accessor :item_id, :user_id, :post_code, :place_id, :municipality, :address, :building, :telephone_number, :token
+
+  with_options presence: true do
+    validates :token, presence: true
```

Comment:
```suggestion
    validates :token
```
5行目で`presence: true`は定義いただいていますので、tokenの後ろの箇所は削除しましょう。